Add signing-key input option

arashi01 opened the pull request:
Currently there is no way to set Scala Steward's `--git-author-signing-key` option. This adds an optional `signing-key` configuration parameter.

alejandrohdezma commented:
Hi! Thanks, this is an awesome improvement. Only thing missing would be to add the new input to the `action.yml` file.

[This](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions) is the reference for GitHub Action's `action.yml` file.

Also, would you mind adding a line or two about this new input to the [GPG](https://github.com/scala-steward-org/scala-steward-action#gpg) section of the README.md? Thanks!

arashi01 commented:
@alejandrohdezma Okay great.

I've added the parameter to `action.yml` as required.

Also added some usage documentation. Let me know if any changes are needed.

alejandrohdezma commented:
It looks great! Thank you for this :)

alejandrohdezma commented:
@arashi01 This is now on `master`, could you give it a go using the commit's SHA (`a57c95bff174c8b29c2f9227505454025fe63889`) as version? If you confirm this works as expected, I will create a new release with the change :)

arashi01 commented:
@alejandrohdezma Okay so it seems to work as expected. Steps taken to test:
- Add correct GPG key ID to Scala Steward workflow file to see if it solved issue we were currently experiencing attempting to sign commits.
- Observed that previous error when enabling GPG signing is no longer there.
- Add incorrect / random text replacing correct key id in Scala Steward workflow file.
- Observed that Scala Steward run failure due to GPG signing error.

alejandrohdezma commented:
That's great! Thanks :)

I just created a new release including this change: https://github.com/scala-steward-org/scala-steward-action/releases/tag/v2.15.0

`v2` tag should also be updated to the same commit.
